api/trace/span_context_test.go

```diff
@@ -176,7 +176,7 @@ func TestSpanContextIsSampled(t *testing.T) {
 			name: "sampled plus unused",
```

Comment:
Is there a better name for this test now that `trace.FlagsUnused` does not exist?

Reply:
I'll take a stab at renaming.